assembl/static2/js/app/utils/translations.js

```diff
@@ -1,6 +1,11 @@
 /* eslint max-len: "off", quotes: ["error", "double"] */
 const Translations = {
   fr: {
+    language: {
```

Comment:
I can understand why these are here, but having each translated manually is unnecessary. The backend has all of these locales in different languages under `LocaleLabel` model. They can be fetched, or put in the jinja file and appended to `translations.js` if need be. Thoughts?

Reply:
Yes, I know, this is temporary. This will be removed. I need a graphql query to get all the locale labels. I'm waiting for your branch for that.